test/Aqua.jl

```diff
@@ -0,0 +1,8 @@
+using Aqua
+
+@testset "Aqua.jl" begin
+    Aqua.test_all(UnROOT;
+    ambiguities = (; broken=true),
+    piracy = (; broken=true)
```

Comment:
- https://github.com/JLTastet/LorentzVectors.jl/pull/11

trying to upstream here, but maybe we should just move everything into LorentzVectorHEP.jl